# -*- coding: utf-8 -*-
# Generated by Django 1.11.4 on 2017-10-09 17:58
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('api', '0003_auto_20171002_2311'),
    ]

    operations = [
        migrations.RenameField(
            model_name='client',
            old_name='bussiness_name',
            new_name='business_name',
        ),
        migrations.RenameField(
            model_name='specialist',
            old_name='bussiness_name',
            new_name='business_name',
        ),
        migrations.AlterField(
            model_name='user',
            name='photo',
            field=models.CharField(default='preview.png', max_length=250, null=True),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
